doc/classes/Node.xml

```diff
@@ -198,7 +198,12 @@
 		<method name="can_process" qualifiers="const">
 			<return type="bool" />
 			<description>
-				Returns [code]true[/code] if the node can process while the scene tree is paused (see [member process_mode]). Always returns [code]true[/code] if the scene tree is not paused, and [code]false[/code] if the node is not in the tree.
+				Returns whether the node can process by checking the value of [member process_mode] and [member SceneTree.paused]. If the node can process, callbacks like [method _process] and [method _physics_process] will be called.
+				If [member process_mode] is set to [constant PROCESS_MODE_PAUSABLE], returns [code]true[/code] when the game is not paused.
+				If [member process_mode] is set to [constant PROCESS_MODE_WHEN_PAUSED], returns [code]true[/code] when the game is paused.
+				If [member process_mode] is set to [constant PROCESS_MODE_ALWAYS], always returns [code]true[/code].
+				If [member process_mode] is set to [constant PROCESS_MODE_DISABLED], always returns [code]false[/code].
+				If [member process_mode] is set to [constant PROCESS_MODE_INHERIT], use the value of its parent's [member process_mode] to determine the result.
```

Comment:
Adding a new line for each process mode sounds beneficial on paper, but in practice it adds a lot of padding of massive padding.

Reply:
Lists are somewhat supported. The online class reference replaces lines starting with "-" with lists, whether it's intentional or not.

<img width=75% src=https://github.com/godotengine/godot/assets/66727710/b21ab033-f2f3-41c2-8824-8a23e6da92a4>

Supporting lists for built-in docs is another question to be figured out (I'm kinda on it), so if you at least prefix these lines with "- " it would at least be more tolerable.

If you can find a way to rewrite these lines without mentioning `process_mode` every time it would also be nice. It's good to be explicit, but doing it 5 times feels excessive.

Reply:
I applied your suggestions. Looking forward to these list changes.